src/_guides/language/coming-from/swift-to-dart.md

```diff
@@ -541,16 +539,15 @@ let str: String? = nil
 let count = str?.count ?? 0
 ```
 
-Additionally, Dart offers a
-[null safe version of the cascade operator][] (`?..`),
-which ignores any operations when
-the target object is `null`.
+In addition, Dart offers a
+[null safe version of the cascade operator][] (`?..`).
+This operator ignores any operations when
+the target expression resolves to `null`.
 Dart also offers the null assignment operator (`??=`),
 which Swift doesn't.
-It allows you to only assign a value to a variable
+This operator permits you to assign a value to a variable
```

Comment:
This changed meaning a bit. The point of this operator is that the assignment is conditional to to when the current value is null:

```
// Assign value to b if b is null; otherwise, b stays the same
b ??= value;
```